app/routes/projects/new.js

```diff
@@ -2,7 +2,7 @@ import Ember from 'ember';
 
 export default Ember.Route.extend({
   model() {
-    return this.store.createRecord('project', {name: this.get('name')});
+    return this.store.createRecord('project', { name });
```

Comment:
can you explain how this works?  looks cool! can you have a comma separated list in that object of just variable names?